src/Sarif.UnitTests/Visitors/SarifVersionOneToCurrentVisitorTests.cs

```diff
@@ -1,194 +1,69 @@
-//            VerifyVersionOneToCurrentTransformationFromResource("OneRunWithBasicInvocation.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_OneRunWithInvocationAndNotifications()
-//        {            
-//            VerifyVersionOneToCurrentTransformationFromResource("OneRunWithInvocationAndNotifications.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_OneRunWithNotificationsButNoInvocations()
-//        {
-//            VerifyVersionOneToCurrentTransformationFromResource("OneRunWithNotificationsButNoInvocations.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
-//        {
-//            VerifyVersionOneToCurrentTransformationFromResource("NotificationExceptionWithStack.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_BasicResult()
-//        {
-//            VerifyVersionOneToCurrentTransformationFromResource("BasicResult.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
-//        {
-//            VerifyVersionOneToCurrentTransformationFromResource("TwoResultsWithFixes.sarif");
-//        }
-
-//        [Fact]
-//        public void SarifTransformerTests_ToCurrent_CodeFlows()
-//        {
-//            VerifyVersionOneToCurrentTransformationFromResource(v1InputResourceName: "CodeFlows.sarif",
-//                                                                v2ExpectedResourceName: "CodeFlows.sarif");
-//        }
-//    }
-//}
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Newtonsoft.Json;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.Sarif.UnitTests.Visitors
+{
+    public class SarifVersionOneToCurrentVisitorTests : FileDiffingTests
+    {
+        public SarifVersionOneToCurrentVisitorTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
+
+        protected override string ConstructTestOutputFromInputResource(string inputResourceName)
+        {
+            string v1LogText = GetResourceText(inputResourceName);
+            SarifLogVersionOne v1Log = JsonConvert.DeserializeObject<SarifLogVersionOne>(v1LogText, SarifTransformerUtilities.JsonSettingsV1Indented);
+            var transformer = new SarifVersionOneToCurrentVisitor();
+            transformer.VisitSarifLogVersionOne(v1Log);
+
+            SarifLog v2Log = transformer.SarifLog;
+            return JsonConvert.SerializeObject(v2Log, SarifTransformerUtilities.JsonSettingsIndented);
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_RestoreFromPropertyBag() => RunTest("RestoreFromPropertyBag.sarif");
```

Comment:
> => RunTest("RestoreFromPropertyBag.sarif"); [](start = 76, length = 44)

Just my 2 cents, I actually don't like the collapsing you've done here. It's hard for my eye to separate the unique name of the test file in these single line constructs (which can get lengthy. I'd find a small method that isolates the call to RunTest much more readable. I assume use of this pattern won't complicate setting a breakpoint. it's certainly true that it will be more difficult to introduce one-off code changes to an individual test with this pattern. final point, it's only 67 lines of length, this file, still small after expansion. So I don't understand what the emphasis on compact code buys us.